src/db/models/Highlight.js

```diff
@@ -0,0 +1,58 @@
+export default (sequelize, DataTypes) => {
+  const Highlight = sequelize.define('Highlight', {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+      allowNull: false,
+    },
+    articleId: {
+      type: DataTypes.UUID,
+      field: 'article_id',
+    },
+    userId: {
+      type: DataTypes.UUID,
+      field: 'user_id',
+    },
+    highlightedText: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      field: 'highlighted_text',
+    },
+    startIndex: {
```

Comment:
Just curious, how do we intend on getting this `startIndex` and `endIndex` on the frontend?

Reply:
We do not have a specific workaround and solution yet but we agree it is the primary way for us to get the exact highlighted text. So we would definitely do more research and come up with the  complete plan of implementation.